.github/actions/upload-gatk-test-results/action.yml

```diff
@@ -34,14 +36,16 @@ runs:
       if: always()
       uses: actions/upload-artifact@v2
       with:
-        name: test-results-${{ matrix.Java }}-${{ matrix.testType }}
+        # ternary expression syntax is workaround found here https://github.com/actions/runner/issues/409
+        name: test-results-${{ inputs.is-docker == 'true' && 'docker-' ||  '' }}${{ matrix.Java }}-${{ matrix.testType }}
```

Comment:
Would it be easier to have the calling tasks just pass in the name?  That way this doesn't have to be aware of docker/java version/general test structures"?